### DISCUSSION TITLE
Change Contact's 2 column layout to single column

### DISCUSSION THREAD

**Discussion by PercyP** [YetiForceCompany/YetiForceCRM#15301] [How to] [answered]
Hi,

I don't know how it has happened but somehow my Contacts form is split in to 2 columns and I cannot figure out how I have done this. 

How do I change it back to a single column layout (see image to understand what i mean). I have checked with Yetiforce stable version, hence sure it is something i have done. 

Is it a setting somewhere?

![changeColums](https://user-images.githubusercontent.com/10598886/136966280-8349ce11-4bb3-4ea9-9d08-f43d54661d8e.png)

Kind regards

**Comment by rskrzypczak** [comment, marked as answer]
Probably you or someone else changes the module type to advanced :)
![image](https://user-images.githubusercontent.com/10221999/136970107-2f2f0683-0af1-4d6b-bf2c-e49888e12bb3.png)
Limit the number of administrators.

**Reply by PercyP** [reply]
Thank you!

That was it. :)
